api/src/main/java/com/epam/pipeline/manager/cluster/performancemonitoring/ResourceMonitoringManager.java

```diff
@@ -133,7 +139,8 @@ static class ResourceMonitoringManagerCore {
                                       final PreferenceManager preferenceManager,
                                       final StopServerlessRunManager stopServerlessRunManager,
                                       final InstanceOfferManager instanceOfferManager,
-                                      final RunStatusManager runStatusManager) {
+                                      final RunStatusManager runStatusManager,
+                                      final ESMonitoringManager monitoringManager) {
```

Comment:
let's use already existing way to request metrics (same as for IdleRuns). For this we need implement a method for `NetworkRequester.java` which now is unimplemented + `AbstractMetricRequester.getRequester` should be expanded to return `NetworkRequester` if such metric is requested